doc/user_guide/user-guide.md

```diff
@@ -0,0 +1,109 @@
+# Python Extension Common User Guide
+
+## Language Container Deployer
+
+An extension would typically use UDF scripts to enable certain custom functionality within a database.
+In most cases, UDF scripts must be backed by a Language Container, that should be installed in the Exasol Database.
+A Script Language Container is a mechanism that allows the installation of the chosen programming language and
```

Comment:
```suggestion
SLCs allow the installation of the chosen programming language and
```

Reply:
An SLC allows ...

Reply:
Yes, often singular is the better choice. Feel free to use your favorite phrasing.